Web/Edubase.Web.UI/Areas/Groups/Controllers/GroupController.cs

```diff
@@ -435,7 +438,7 @@ public async Task<ActionResult> EditDetails(GroupEditorViewModel viewModel)
                 var dto = CreateSaveDto(viewModel).Group;
                 var changes = await _groupReadService.GetModelChangesAsync(dto, User);
 
-                if (changes.Any() && viewModel.GroupTypeId.OneOfThese(GT.SingleacademyTrust, GT.MultiacademyTrust) && !viewModel.ChangesAcknowledged)
+                if (changes.Any() && GroupEditorViewModelRulesHandler.ShowChangesReviewScreen(viewModel, User))
```

Comment:
Minor naming query:

To me, `GroupEditorViewModelRulesHandler.ShowChangesReviewScreen(...)` is ambiguous in what side-effects (if any) it has -- especially if we consider methods to be for behaviour rather than data.

- e.g., does it actually "show" the changes screen?

Personally I would name it to `IsX` / `CanX` / `MayX` or similar so that it's easier to see that it returns a boolean value, but I also appreciate naming is hard / a highly debated topic so I'll not push on this :)

Reply:
@CathLass any thoughts on this Catherine was thinking would let you do this as this is your change.

Reply:
Catherine renamed to MustShowChangesReviewScreen.
Happy that is a good name.  Marking as resolved.